airbyte-integrations/connectors/source-recurly/source_recurly/streams.py

```diff
@@ -0,0 +1,219 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+from abc import abstractmethod
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional, Union
+
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.sources.streams import Stream
+from recurly import Client
+
+
+class BaseRecurlyStream(Stream):
+    DEFAULT_PRIMARY_KEY = "id"
+    DEFAULT_CURSOR = "updated_at"
+    DEFAULT_SORT_KEY = "updated_at"
+
+    BEGIN_TIME_PARAM = "begin_time"
+
+    def __init__(self, client: Client, begin_time: str = None, **kwargs):
+        super(Stream, self).__init__(**kwargs)
+
+        self._client = client
+        self.begin_time = begin_time
+
+    @property
+    @abstractmethod
+    def name(self):
+        """
+        The name of the Recurly resource. All children inherits from this class should set the resource name
```

Comment:
grammar

Reply:
Fixed in https://github.com/mohamagdy/airbyte/pull/1/commits/54786be7856f1fb79dfe0306785fb3e18f00cef7